decide/utils.test.cpp

```diff
@@ -8,12 +8,27 @@ void test_angle_90_degree() {
   assert((angle - pi / 4.0) < 1e-10);
 }
 
+void test_angle_between_0_2pi() {
+  double angle1 = angle_between_points(Coordinate(0, 1), Coordinate(0, 0),
+                                       Coordinate(1, 0));
+  double angle2 = angle_between_points(Coordinate(1, 1), Coordinate(0, 0),
+                                       Coordinate(1, 0));
+  double angle3 = angle_between_points(Coordinate(-1, -1), Coordinate(0, 0),
+                                       Coordinate(1, 0));
+
+  assert((0 <= angle1) && (angle1 < 2 * pi));
```

Comment:
Should we perhaps test their numerical values instead of just testing that the values given are allowed?

Reply:
That would indeed be even more useful.  Will see if I can add that later.